athlinks-ontology.ttl

```diff
@@ -86,6 +86,24 @@
 	rdfs:comment "The numeric order of an object among siblings of the same property or type."@en ;
 	rdfs:label "sibling ordering"@en .
 
+:steps a owl:DatatypeProperty ;
+  rdfs:domain :AthleticEventCompetitor ;
+  rdfs:range xsd:integer;
+  rdfs:comment "The number of steps an athlete took in an athletic event"@en ;
+  rdfs:label "steps"@en .
```

Comment:
Shouldn't this be added under the RaceMetrics object?

Reply:
Sorry, I meant to type `RacerMetrics`.

````
:RacerMetrics a owl:Class ;
  rdfs:subClassOf schema:DataFeed;
  rdfs:comment "A set of data metrics for a given athlete. The about property should be used to refer to the athlete (schema:Person) about whom the metrics are computed."@en ;
  rdfs:label "Racer metrics"@en .
````

Andy did work on this, so you might want to discuss implementation issues here with him.

Reply:
@LPerkins3 I don't know the full scope of the steps requirement, but I think it makes sense to have it be a `result` on an `Action` (analogous to a time result), but also to have a `steps` metric on the `RacerMetrics` object, for aggregating across events.